horovod/common/util.py

```diff
@@ -152,9 +152,7 @@ def gloo_built(verbose=False):
             ext_base_name, built_fn, 'built with Gloo', verbose)
         if result is not None:
             return result
-    raise RuntimeError('Failed to determine if Gloo support has been built. '
```

Comment:
Better print a message in log for debugging later.

Reply:
fixed.